### PR TITLE
golang example

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by aw185176 — APPROVED** [review]

File: go/sign/access_key_singer_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func ExampleAccessKeyHTTPSigner_Sign() {
+	req, _ := http.NewRequest("GET",
+		"https://gateway-staging.ncrcloud.com/site/sites/find-nearby/88.05,46.25?radius=10000",
```
Comment (cc185283) [inline go/sign/access_key_singer_test.go]: I would externalize this URL to not have to require a code change if the value is ever changed in the future

**Review by aa250816 — APPROVED** [review]

**Review by ss185427 — COMMENTED** [review]
access_key_si**ng**er_test.go  should probably be renamed to access_key_si**gn**er_test.go

File: go/sign/access_key_singer_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func ExampleAccessKeyHTTPSigner_Sign() {
```
Comment (ss185427) [inline go/sign/access_key_singer_test.go]: Without being aware of the requirements, just wanted to mention changing ExampleAccessKeyHTTPSigner_Sign to TestAccessKeyHTTPSigner_Sign() (and import "testing") will help in using 'go test' command for unit testing.

Comment (ghost) [reply go/sign/access_key_singer_test.go]: > access_key_si**ng**er_test.go should probably be renamed to access_key_si**gn**er_test.go

Good catch. I'll make the change

File: go/sign/access_key_signer_test.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	req.Header.Add("Content-Type", "application/json")
+
+	// Replace the empty string with your shared key
+	sharedKey := ""
+	// Replace the empty string with your secret key
+	secretKey := ""
+	s, _ := NewAccessKeyHTTPSigner(sharedKey, secretKey)
+	req, err := s.Sign(req)
+	if err != nil {
+		log.Fatal(err)
```
Comment (ss185427) [inline go/sign/access_key_signer_test.go]: Prefer t.Fatal / t.Fatalf to indicate test case failure and end test case execution.

Comment (ghost) [reply go/sign/access_key_signer_test.go]: Good catch I'll update it

**Comment by kg185155** [comment]
Test

**Pull request merged**